images/reviewer/Dockerfile

```
@@ -0,0 +1,16 @@
+FROM ubuntu:20.04
+ARG DEBIAN_FRONTEND=noninteractive
+
+# hadolint ignore=DL3008
+RUN apt-get update && apt-get -y install wget && \
+wget --progress=dot:giga https://github.com/Illumina/REViewer/releases/download/v0.2.7/REViewer-v0.2.7-linux_x86_64.gz && \
+gunzip REViewer-v0.2.7-linux_x86_64.gz && \
+chmod +x REViewer-v0.2.7-linux_x86_64 && \
+apt-get clean && \
+rm -rf /var/lib/apt/lists/*
+
+# Clean up
+RUN rm -rf /tmp/* \
+    /var/tmp/* \
+    /var/cache/apt/*
```

Can you just join these runs together to save a layer - and I've added the gzipped file.

```suggestion
rm -rf /var/lib/apt/lists/* \
    /tmp/* \
    /var/tmp/* \
    /var/cache/apt/* \
    REViewer-v0.2.7-linux_x86_64.gz
```